src/pages/showmodules/index.tsx

```diff
@@ -28,29 +28,49 @@ function ShowModules() {
   }
 
   // Calling the hook which contains the state and return the data of items
-  const {items, searchValue, orderItems, filterStatus} = useGetModules(titleModule);
-  console.log(items)
+  const { items, searchValue, orderItems, filterStatus } =
+    useGetModules(titleModule);
+
   return (
     <>
       <Head>
         <title>Studio Ghibli Backoffice - Administrator - Show Modules</title>
-    </Head>
-    <Header/>
-    <Title title={titleModule ? titleModule : "Show Module"}/>
-    <section className="flex flex-wrap items-end justify-center min-h-full gap-6 px-4 py-16 sm:px-6 lg:px-8">
```

Comment:
We need those classnames to make responsive the filters container

Reply:
Done.

Reply:
Mobile:

https://user-images.githubusercontent.com/17114826/178569935-2646f5ac-41fd-4710-930b-b88b66fbcc53.mov


Desktop:


https://user-images.githubusercontent.com/17114826/178569972-bb7e8150-5e55-4a92-92fb-e5fbfd7c7315.mov